kotlin/vector-tiled-layer-custom-style/src/main/res/layout/spinner_item.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="10dp"
+    android:orientation="horizontal">
+
+
+    <View
+        android:id="@+id/color_view"
```

Comment:
ids are usually camelCase, and layout file names snake_case in Android...don't ask me why 🤣